radix-engine/src/model/epoch_manager/executables.rs

```diff
@@ -64,11 +71,19 @@ impl Executor for EpochManagerCreateInvocation {
             rounds_per_epoch: self.rounds_per_epoch,
         };
 
-        let mut validator_set = BTreeSet::new();
+        let mut validator_set = BTreeMap::new();
 
-        for key in self.validator_set {
-            let address = EpochManager::create_validator(global_node_id.into(), key, api)?;
-            validator_set.insert(Validator { address, key });
+        for (key, bucket) in self.validator_set {
+            let stake = Decimal::one();
```

Comment:
Is this  `stake_units` or `staked_xrd_amount`? It doesn't seem to be set according to the bucket.

Reply:
ooo yup good catch